docs/interactive.md

```diff
@@ -0,0 +1,128 @@
+# Interactive Python Console
+
+Tomviz provides a Python console and interactive API that can be used to build up
+and interact with pipelines from within Python. The Python console can be enabled
+using the View menu.
+
+
+# Loading a state file
+
+```python
+>>>
+Python 3.7.5 (default, Nov 20 2019, 09:21:52)
+[GCC 9.2.1 20191008] on linux
+>>> from tomviz import state
+>>> state.load('/home/cjh/tmp/test.tvsm')
```

Comment:
Should we just use /tmp/test.tvsm here too, keep it generic, any user can write in /tmp, kinda 50/50 as new users may just want to copy/paste. It is a shame that ~/ doesn't work...